protocol/query/method/message/data/data.json

```diff
@@ -1,47 +1,47 @@
 {
-	"$schema": "http://json-schema.org/draft-07/schema#",
-	"$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/query/method/message/data/data.json",
-	"description": "Match one type of data",
-	"type": "object",
-	"oneOf": [
-		{
-			"$ref": "dataCastVote.json"
-		},
-		{
-			"$ref": "dataCloseRollCall.json"
-		},
-		{
-			"$ref": "dataCreateLao.json"
-		},
-		{
-			"$ref": "dataCreateMeeting.json"
-		},
-		{
-			"$ref": "dataCreateRollCall.json"
-		},
-		{
-			"$ref": "dataElectionEnd.json"
-		},
-		{
-			"$ref": "dataElectionResult.json"
-		},
-		{
-			"$ref": "dataElectionSetup.json"
-		},
-		{
-			"$ref": "dataOpenRollCall.json"
-		},
-		{
-			"$ref": "dataStateLao.json"
-		},
-		{
-			"$ref": "dataStateMeeting.json"
-		},
-		{
-			"$ref": "dataUpdateLao.json"
-		},
-		{
-			"$ref": "dataWitnessMessage.json"
-		}
-	]
-}
\ No newline at end of file
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/query/method/message/data/data.json",
+    "description": "Match one type of data",
+    "type": "object",
+    "oneOf": [
```

Comment:
would it make sense to factor this out in separate subdirectories, e.g. at this level 
```
        {
            "$ref": "election/index.json"
        },
        {
            "$ref": "lao/index.json"
        },
```
and so on, while in `election/index.json` there would be :
```
        {
            "$ref": "CastVote.json"
        },
        {
            "$ref": "EndElection.json"
        },
        {
            "$ref": "ResultElection.json"
        },
        {
            "$ref": "SetupElection.json"
        },
```
?